src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+
+  if (days < 3) {
+    return dailyRent * days;
```

Comment:
please do this calculation once 
currently it is repeated 3 times

Reply:
Done!